notebooks/vertex_genai/solutions/vertex_llm_evaluation.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">for details in job.task_details:
&nbsp; &nbsp; if details.task_name == "model-evaluation-text-generation-pairwise":
&nbsp; &nbsp; &nbsp; &nbsp; break
</pre><p> </p><p>Same as the comment above.</p>

<pre class="ql-syntax" spellcheck="false">online_eval_task = [task for task in job.task_details if task.task_name=="online-evaluation-pairwise"][0]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>That's nicer. Thanks. Done!</p>